### ISSUE TITLE
loadJSONObject and loadJSONArray are missing

### ISSUE THREAD

**Issue by ekeneijeoma** [processing/processing-android#189]
I see that the JSONObject and JSONArray classes are there but loadJSONObject and loadJSONArray aren't accessible. In the meantime I'm just doing this

```
String[] ls = loadStrings("file.json");
JSONObject jo = JSONObject.parse(join(ls,""));
```

**Comment by suheb** [comment]
Fix #190

**Comment by codeanticode** [comment]
merged fix from @suheb
